src/line-chart/line-chart-grouped.js

```diff
@@ -3,17 +3,27 @@ import ChartGrouped from '../chart/chart-grouped'
 
 class LineChartGrouped extends ChartGrouped {
     createPaths({ data, x, y }) {
-        const { curve } = this.props
-
-        const lines = data.map((line) =>
-            shape
+        const { curve, continuousLine } = this.props
+        let line
+        if (continuousLine) {
+            line = shape
+                .line()
+                .x((d) => x(d.x))
+                .y((d) => y(d.y))
+                .curve(curve)(data.filter((item) => item.y))
```

Comment:
I think this should look like below, since `data` is array of `line`.

```suggestion
            lines = data.map((line) =>
                shape
                .line()
                .x((d) => x(d.x))
                .y((d) => y(d.y))
                .curve(curve)(line.filter((item) => item.y))
```